src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[fromFormat.length - 1];
+  const oldDate = date.split(oldSeparator);
+  const newSeparator = toFormat[fromFormat.length - 1];
+  const newDate = new Array(oldDate.length);
+
+  for (let i = 0; i < oldDate.length; i++) {
+    if (fromFormat[i][0] === 'Y' && toFormat.indexOf(fromFormat[i]) < 0) {
```

Comment:
smart solution 👍

Reply:
fromFormat[i][0] === 'Y'  - remove this part of code. It's not needed.